### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by Mikisepic** [StonksCo/beer-stonks#25] [General]
Discussions regarding deployment of the application

**Comment by Mikisepic** [comment]
Vercel Teams are needed in order to deploy repos from GitHub organizations. @Jok3r182  any ideas for alternatives in case needed?

**Comment by wolftherai** [comment]
https://www.wpbeginner.com/showcase/best-free-website-hosting-compared/

**Comment by Mikisepic** [comment]
[Azure](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-to-azure-button)
[Firebase](https://cloud.google.com/build/docs/deploying-builds/deploy-firebase)

**Reply by Jok3r182** [reply]
Wont azure be expensive? 100eur for 1 month?
Firebase is a good option.

**Reply by Mikisepic** [reply]
100????? 😮 : damn

**Reply by Mikisepic** [reply]
Haven't tried deploying to firebase tho, only its database

**Reply by Jok3r182** [reply]
100 for a free trial, and then they ask you to provide payment data for further usage. I’ve done it once until trial ended, it cost me around 3-7eu for a day.

**Reply by Jok3r182** [reply]
Firebase is free and a good alternative, also is an easy option to setup, has a realtime database, so we wont gonna need a mongodb.

**Reply by Jok3r182** [reply]
I mean there is a hostinger 2.5eur a month, also they give an option for a free domain.

**Reply by Mikisepic** [reply]
> Firebase is free and a good alternative, also is an easy option to setup, has a realtime database, so we wont gonna need a mongodb.

Yep that's a great option as well
